lib/features/allow_ptrace.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * This feature adjusts the AppArmor profile to allow ptrace within a container.
+ * All of the interesting action takes place in `lib/task.js`.
+ */
+
+export default class AllowPtrace {
+  constructor() {
+    this.featureName = 'allowPtrace';
```

Comment:
Is there any harm in not limiting this feature by a scope?  Currently all features are allowed unless explicitly set to check a scope in this class.